components/MapList/index.js

```diff
@@ -177,15 +176,17 @@ function MapList({
                   hasSelectedFeature={!!selectedFeature}
                   featureCounts={featureCounts}
                 />
-                <div className="px-3" style={{ overflowY: "scroll" }}>
+                {searchedGeojson.features.length ? <div className="px-3" style={{ overflowY: "scroll" }}>
                   <List
                     geojson={searchedGeojson}
                     mapRef={mapRef}
                     setSelectedFeature={setSelectedFeature}
                     ListItemContent={ListItemContent}
                   />
-                </div>
-              </div>
+                </div> :
+                  <div className="p-3">Signal system operating normally.</div>
+                }
+              </div> 
```

Comment:
I don't think this approach is going to work, because this is component is used by other dashboards, e.g. the cameras dashboard:
<img width="276" alt="Screenshot 2024-05-29 at 10 59 37 AM" src="https://github.com/cityofaustin/atd-data-and-performance/assets/14793120/6e682015-0705-4c6c-a4a9-4864e5c61b30">

The other problem is that this message displays if you search the list when there is a signal outage:
<img width="556" alt="Screenshot 2024-05-29 at 11 00 26 AM" src="https://github.com/cityofaustin/atd-data-and-performance/assets/14793120/886f12d6-a360-43dc-8c9e-59d5ad1dd04b">

I guess one more problem is that *i think* this "everything is ok" message will display if there is no data returned from the Socrata query—e.g. if there is a socrata outage. just looking very quickly at the code, i'm not sure we currently display an error message under any circumstances. 

From a styling perspective, it would be nice if the message rendered inside a list item with the same padding as other list items. Maybe using a green alert banner or something besides just plain text?

Reply:
i did not think to check if the component is used by other dashboards! thank you for pointing that out. and on your second and third point you're totally right i need to use a more robust way of checking why there might not be data returned from socrata. and i can definitely add styling to the message. @johnclary should we break these up into two issues (one to remove comm outages and one to display a message stating everything is working) or should i rework this?

Reply:
I know you asked John, but my opinion is to rework this

Reply:
i defer to Chia and Christina! 👍 🙏